scenarios/arcaflow/kill-pod/workflow.yaml

```diff
@@ -0,0 +1,42 @@
+input:
+  root: RootObject
+  objects:
+    RootObject:
+      id: RootObject
+      properties:
+        kubeconfig:
+          display:
+            description: The complete kubeconfig file as a string
+            name: Kubeconfig file contents
+          type:
+            type_id: string
+          required: true
+        namespace_pattern:
+          display:
+            description: The namespace where the pod resides
+            name: namespace pattern
+          type:
+            type_id: string
+        name_pattern:
+          display:
+            description: the pod name to kill
+            name: name pattern
+          type:
+            type_id: string
+        kill:
+          display:
+            description: the number of pods to kill
+            name: kill
+          type:
+            type_id: integer
+steps:
+  kill_pod:
+    plugin: quay.io/tsebastiani/arcaflow/kill-pod:latest
```

Comment:
We might want to move this image to redhat-chaos quay org - https://quay.io/organization/redhat-chaos.